ebos/eclgenericoutputblackoilmodule.cc

```diff
@@ -1075,6 +1150,104 @@ doAllocBuffers(unsigned bufferSize,
         rstKeywords["BG"] = 0;
         invB_[gasPhaseIdx].resize(bufferSize, 0.0);
     }
+    
+    enableFlows_ = false;
+    enableFlowsn_ = false;
+    if (rstKeywords["FLOWS"] > 0) {
+        rstKeywords["FLOWS"] = 0;
+        enableFlows_ = true;
+        if (FluidSystem::phaseIsActive(gasPhaseIdx)){
+            if (this->eclState_.gridDims()[0] > 1)
```

Comment:
This test should be `> 0` not ` > 1`, as we should output a J+ value even if we have only one cell in the J direction. Postprocessors such as ResInsight expect this. You could test this with a 10 x 1 x 10 grid.